.github/workflows/scripts/check_commit.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I'd probably move this to .github/scripts/.

Reply:
I'd just move all the github-specific scripts to .github/scripts and dump .github/workflows/scripts.

Reply:
```.github``` isn't really a CI directory though, it's a catch-all for Github config files.  ```.github/workspaces``` is the directory specific to CI stuff.